python/ray/experimental/channel.py

```diff
@@ -159,3 +160,17 @@ def end_read(self):
         self._worker.core_worker.experimental_mutable_object_read_release(
             [self._base_ref]
         )
+
+    def close(self) -> None:
+        """
+        Close this channel by setting the error bit on the object.
+
+        Does not block.
```

Comment:
Should add a comment here that any values that are currently in the channel may not be able to be read after close() is called.